### PR TITLE
style: update theme to match MITxOnline Site

### PR DESCRIPTION
**Pull request by arslanashraf7**
## Related Ticket:
#3 
🚧 Rebased on #2 

## What this PR Does?
- Update the Zendesk theme to match the MITxOnline site
- Adds Contact Us block
- Adds header design changes
- Adds tweaks for updating hero design
- Updates the site logo
- Updates the header

## How to test

- Make sure that all the checkboxes mentioned in #3 should be completed in this PR
- You can visit https://mitxonline-test-theme.zendesk.com/hc/en-us to check this theme in a test instance.

## Screenshots
**Desktop**
screencapture-mitxonline-test-theme-zendesk-hc-en-us-2021-12-15-19_03_45

**Mobile**
screencapture-mitxonline-test-theme-zendesk-hc-en-us-2021-12-15-19_04_09

### CODE REVIEW

**Comment by arslanashraf7** [comment]
@briangrossman Could you take a look at the screenshots attached in the PR description and let me know if they look good. Also, You can check this theme in the test instance (https://mitxonline-test-theme.zendesk.com/hc/en-us).

**Comment by briangrossman** [comment]
@arslanashraf7 This looks great. Nice work. Thanks for setting up the demo site, too!

It looks like you took care of everything. 👍

**Comment by arslanashraf7** [comment]
@umarmughal824 On the note of reviewing the PR I just wanted to add that if you could review this PR you won't need to review the other one related to footer (#3 ) since it is already rebased on that & includes all the changes of the footer as well.

File: templates/footer.hbs [hunk 1]
```diff
@@ -13,25 +13,29 @@
...
-    <div class="company-info">
-        <img src="{{settings.mit_footer_logo}}">
-        <address>
-          {{settings.mit_address}}
-        </address>
+
+  <div class="container">
+      <div class="footer-top">
+        <div class="mit-info">
+          <img src="{{settings.mit_footer_logo}}" width="78" height="41" alt="MIT" loading="lazy"/>
```
Comment (umarmughal824) [inline templates/footer.hbs:20]: please define the styling in `.scss` file. My concern here is for width and height

**Review by umarmughal824 — CHANGES_REQUESTED** [review]
Few concerns

- it seems very inconsistent that at some points we are using hexadecimal color and others using variables defined against hexadecimal values. We should defined variable against each hexadecimal values
- some classes are using #000 as a color but we have defined a variable against it already `mit-header-link-color` that should be used
- in some media queries 100% for screen sizes is being used even though we have define a variable against it in variables `max-width-container` that should be used
- even though we could define all screen sizes here in variable against 1024, 768 and 576 and used these variable. I could see some variables are already defined so better to use them
- it's better to use variables for all screen sizes its also inconsistent at some places you are using variables and and others using hard coded values so better to use variables

File: style.css [hunk 2]
```diff
@@ -840,62 +921,32 @@ ul {
...
 
-.footer .contact-block:before {
-  position: absolute;
-  left: 0;
-  bottom: 0;
-  right: 0;
-  content: '';
-  height: 30px;
-  background: rgba(0, 0, 0, 0.3);
+@media (max-width: 576px) {
```
[the remark this answers is not in the document]
Comment (arslanashraf7) [reply style.css]: Updated & Replaced with mobile.

File: style.css [hunk 3]
```diff
@@ -1057,6 +1108,9 @@ ul {
   .hero .search input[type="search"] {
     width: calc(100% - 150px);
   }
+  .hero .search input[type="search"] input#query {
+    color: #000000;
```
[the remark this answers is not in the document]
Comment (arslanashraf7) [reply style.css:1111]: Updated

File: templates/footer.hbs (hunk 1 above)
Comment (arslanashraf7) [reply templates/footer.hbs]: Updated

**Comment by arslanashraf7** [comment]
@umarmughal824 I have addressed your feedback and it's ready for review again.

File: styles/_header.scss [hunk 4]
```diff
@@ -4,29 +4,30 @@ $header-height: 71px;
...
+    text-transform: none;
+    font-weight: $font-weight-semibold;
   }
 
   @include max-width-container;
   position: relative;
   align-items: center;
   display: flex;
   justify-content: space-between;
+  padding: 5px 2% 5px 2%;
```
Comment (umarmughal824) [inline styles/_header.scss:16]: This should have been written
`padding: 5px 2%`

File: style.css [hunk 5]
```diff
@@ -133,12 +133,12 @@ ul {
...
 
 .error-page {
-  max-width: 1160px;
+  max-width: 100%;
   margin: 0 auto;
   padding: 0 5%;
 }
 
-@media (min-width: 1160px) {
+@media (min-width: 100%) {
```
Comment (umarmughal824) [inline style.css:141]: At least in all media queries we could use $max-width-container variable against 100%

File: style.css [hunk 6]
```diff
@@ -430,41 +429,42 @@ ul {
...
 .header {
-  max-width: 1160px;
+  max-width: 100%;
   margin: 0 auto;
   padding: 0 5%;
   position: relative;
   align-items: center;
   display: flex;
   justify-content: space-between;
+  padding: 5px 2% 5px 2%;
```
Comment (umarmughal824) [inline style.css:439]: Use `5px 2%`

File: style.css [hunk 7]
```diff
@@ -430,41 +429,42 @@ ul {
...
 }
 
 .header .submit-a-request {
-  text-transform: uppercase;
+  text-transform: none;
+  font-weight: 600;
 }
 
-@media (min-width: 1160px) {
+@media (min-width: 100%) {
```
Comment (umarmughal824) [inline style.css:449]: Same concern

File: style.css [hunk 8]
```diff
@@ -430,41 +429,42 @@ ul {
...
   .header {
     padding: 0;
     width: 90%;
   }
 }
 
 .support-center {
-  color: $site_theme_color;
+  color: #000000;
```
Comment (umarmughal824) [inline style.css:457]: Better to define a variable against it and use that everywhere in that file

File: style.css [hunk 9]
```diff
@@ -537,7 +538,7 @@ ul {
 
 .nav-wrapper a {
   border: 0;
-  color: $link_color;
+  color: #000000;
```
Comment (umarmughal824) [inline style.css:541]: Same concern

File: style.css [hunk 10]
```diff
@@ -557,7 +558,7 @@ ul {
 
 .nav-wrapper a:hover, .nav-wrapper a:focus, .nav-wrapper a:active {
   background-color: transparent;
-  color: $link_color;
+  color: #000000;
```
Comment (umarmughal824) [inline style.css:561]: Same

File: style.css [hunk 11]
```diff
@@ -683,155 +684,234 @@ ul {
 }
 
 /***** Footer *****/
+.container {
+  padding: 0 32px 0 32px !important;
```
Comment (umarmughal824) [inline style.css:688]: `padding: 0 32px`

File: style.css [hunk 12]
```diff
@@ -683,155 +684,234 @@ ul {
...
 
-@media (min-width: 1160px) {
-  .footer-inner {
-    padding: 0;
-    width: 90%;
+.footer .contact-block {
+  text-align: center;
+  padding: 30px 0 80px;
+  position: relative;
+  background: #f5f5f5;
```
Comment (umarmughal824) [inline style.css:718]: Don't we have any variable against it?

File: style.css [hunk 13]
```diff
@@ -683,155 +684,234 @@ ul {
...
-  padding-top: 20px !important;
-  padding-bottom: 20px !important;
+.footer .contact-block:after {
+  position: absolute;
+  left: 0;
+  top: 0;
+  right: 0;
+  content: '';
+  height: 15px;
+  background: #c2c2c2;
```
Comment (umarmughal824) [inline style.css:735]: Same

File: style.css [hunk 14]
```diff
@@ -683,155 +684,234 @@ ul {
...
-    width: 90%;
+.footer .contact-block {
+  text-align: center;
+  padding: 30px 0 80px;
+  position: relative;
+  background: #f5f5f5;
+  color: #000000;
+}
+
+@media (max-width: 768px) {
```
Comment (umarmughal824) [inline style.css:722]: Better to use tablet variable here

File: style.css [hunk 15]
```diff
@@ -683,155 +684,234 @@ ul {
...
+  content: '';
+  display: block;
+}
+
+.footer .footer-bottom {
+  padding-top: 54px;
+  flex-direction: row-reverse;
+}
+
+@media (max-width: 768px) {
```
Comment (umarmughal824) [inline style.css:787]: Same

File: style.css [hunk 16]
```diff
@@ -683,155 +684,234 @@ ul {
...
+  display: inline-block;
+  color: white;
+  min-width: 180px;
+  padding: 10px 30px;
   text-align: center;
-  padding: 0 0 20px;
+  font-size: 24px;
+  line-height: 30px;
+  font-weight: 500;
+  background: #006b9c;
```
Comment (umarmughal824) [inline style.css:768]: Same

File: style.css [hunk 17]
```diff
@@ -683,155 +684,234 @@ ul {
...
+  color: white;
+}
+
+.footer .mit-info {
+  font-size: 14px;
+  line-height: 24px;
+  padding: 0 0 32px 0;
+}
+
+@media (min-width: 1024px) {
```
Comment (umarmughal824) [inline style.css:803]: Same

File: style.css [hunk 18]
```diff
@@ -683,155 +684,234 @@ ul {
...
-.footer .company-info img {
+.footer .mit-info img {
   width: 78px;
+  height: 41px;
   margin: 0 auto 15px;
 }
 
-@media (min-width: 768px) {
-  .footer .company-info img {
+@media (min-width: 1024px) {
```
Comment (umarmughal824) [inline style.css:817]: Same

File: style.css [hunk 19]
```diff
@@ -683,155 +684,234 @@ ul {
...
+  .footer .mit-info img {
     margin: 5px 15px 0 0;
     float: left;
   }
 }
 
-@media (min-width: 768px) {
-  .footer .company-info img {
-    margin: 5px 25px 0 0;
+@media (min-width: 1024px) {
```
Comment (umarmughal824) [inline style.css:824]: Here too

File: style.css [hunk 20]
```diff
@@ -683,155 +684,234 @@ ul {
...
+  text-align: left;
+}
+
+.footer .right-info {
+  text-align: center;
+  padding: 0 0 20px;
+  display: none;
+}
+
+@media (min-width: 1024px) {
```
Comment (umarmughal824) [inline style.css:848]: Here

File: style.css [hunk 21]
```diff
@@ -683,155 +684,234 @@ ul {
...
 .footer .footer-nav {
-  margin: 0;
   padding: 0;
+  margin: 0;
   list-style: none;
-  text-align: center;
 }
 
-@media (min-width: 768px) {
+@media (min-width: 1024px) {
```
Comment (umarmughal824) [inline style.css:862]: Here

File: style.css [hunk 22]
```diff
@@ -683,155 +684,234 @@ ul {
...
   vertical-align: top;
-  font-size: 20px;
+  margin: 0 0 0 10px;
+  font-size: 18px;
   line-height: 28px;
-  margin: 0 15px;
 }
 
-@media (min-width: 768px) {
+@media (min-width: 1024px) {
```
Comment (umarmughal824) [inline style.css:876]: Here

File: style.css [hunk 23]
```diff
@@ -683,155 +684,234 @@ ul {
...
+.footer .footer-sub-nav {
+  margin: 0 0 5px;
+  list-style: none;
+  text-align: left;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 0 0 20px 0;
 }
 
 @media (min-width: 768px) {
```
Comment (umarmughal824) [inline style.css:895]: Here

File: style.css [hunk 24]
```diff
@@ -840,62 +920,32 @@ ul {
...
 
-.footer .contact-block:before {
-  position: absolute;
-  left: 0;
-  bottom: 0;
-  right: 0;
-  content: '';
-  height: 30px;
-  background: rgba(0, 0, 0, 0.3);
+@media (min-width: 768px) {
```
Comment (umarmughal824) [inline style.css:929]: Here and where ever you find that in the same file

File: style.css [hunk 25]
```diff
@@ -840,62 +920,32 @@ ul {
...
+    padding-bottom: 10px;
+  }
 }
 
-.footer .contact-block h1 {
-  font-size: 35px;
-  line-height: 44px;
-  font-weight: 600;
-  margin: 0 0 20px;
+@media (min-width: 1024px) {
```
Comment (umarmughal824) [inline style.css:935]: Here

File: style.css [hunk 26]
```diff
@@ -840,62 +920,32 @@ ul {
...
-  color: white;
-  min-width: 180px;
-  padding: 10px 10px;
-  text-align: center;
-  font-size: 24px;
-  line-height: 30px;
-  font-weight: 500;
-  background: #006b9c;
-  border-radius: 5px;
+@media (min-width: 1024px) {
```
Comment (umarmughal824) [inline style.css:945]: Here

File: style.css [hunk 27]
```diff
@@ -1032,13 +1082,13 @@ ul {
 .hero .search input[type="submit"] {
   float: right;
   width: 100px;
-  background: #355B6B;
+  background: #ae1f34;
```
Comment (umarmughal824) [inline style.css:1085]: Here

File: style.css [hunk 28]
```diff
@@ -1057,6 +1107,9 @@ ul {
   .hero .search input[type="search"] {
     width: calc(100% - 150px);
   }
+  .hero .search input[type="search"] input#query {
+    color: #000000;
```
Comment (umarmughal824) [inline style.css:1111]: Here

File: style.css [hunk 29]
```diff
@@ -1197,11 +1250,14 @@ ul {
 
 .blocks-item-internal a {
   color: $text_color;
+  background-color: $brand_color;
 }
 
 .blocks-item-link {
   color: $brand_color;
+  background-color: #ae1f34;
```
Comment (umarmughal824) [inline style.css:1258]: Here

**Review by umarmughal824 — CHANGES_REQUESTED** [review]

**Review by umarmughal824 — APPROVED** [review]
👍 , Just a few minor concerns feel free to merge after that.

Make sure you have tested it out after deploying too

**Comment by arslanashraf7** [comment]
Feedback addressed & the final changes have also been deployed at https://mitxonline-test-theme.zendesk.com/hc/en-us/. I'm going to merge this PR.

**Pull request merged**